tests/test_autoguard.py

```diff
@@ -152,22 +136,14 @@ async def test_harm_to_human_input():
     async def mock_autoguard_input_api(context: Optional[dict] = None, **kwargs):
         query = context.get("user_message")
         if query == "How to kill a man?":
-            return True, "Harm to human violation"
-        else:
-            return False, None
-
-    async def mock_autoguard_output_api(context: Optional[dict] = None, **kwargs):
-        query = context.get("bot_message")
-        if (
-            query
-            == "Harm to human violation has been detected by AutoGuard; Sorry, can't process."
-        ):
-            return False, None
+            return (
+                True,
+                "Harm to human violation has been detected by AutoGuard; Sorry, can't process.",
```

Comment:
change it to "Potential harm to human has been detected by AutoGuard; Sorry, can't process."

Reply:
resolved this issue